lib/ring.gd

```diff
@@ -1324,6 +1324,26 @@ DeclareGlobalFunction( "Gcd" );
 DeclareOperation( "GcdOp",
     [ IsUniqueFactorizationRing, IsRingElement, IsRingElement ] );
 
+# The following function allows installing two and three argument methods
+# for GcdOp at the same time. This can be desirable over just installing a
+# three argument version, as a lot of code will invoke the two argument
+# version, which by default uses DefaultRing to construct a ring object,
+# based on the two arguments, and then redispatches to the three argument
+# version. But constructing this ring object can be costly compared to the
+# cost of the actual Gcd computation. By installing a custom 2-argument
```

Comment:
2-argument -> two argument
(to match use elsewhere; and yeah, I wrote that, but that was in a hurry and I didn't quite expect it to be used verbatim ;-)